packages/react/scripts/build.js

```diff
@@ -28,7 +28,7 @@ try {
   exec(`${babelPath} src --quiet -d es --ignore "${ignoreGlobs}"`, {
     BABEL_ENV: 'es',
   });
-  exec(`${babelPath} src --quiet -d lib --ignore "${ignoreGlobs}"`, {
+  exec(`${babelPath} src -d lib --ignore "${ignoreGlobs}"`, {
```

Comment:
Out of curiosity, what's the reason for this change?

Reply:
It was unintentional. Good spot, thanks!